core/src/main/resources/hudson/AboutJenkins/index.jelly

```diff
@@ -24,8 +24,10 @@ THE SOFTWARE.
 
 <!-- About Jenkins page -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
-  <l:layout type="one-column" permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%about(app.VERSION)}">
+<j:jelly xmlns:st="jelly:stapler" xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
```

Comment:
Would it be worth having a new component (e.g. SettingsItemLayout.jelly) rather than `layout.jelly` to wrap pages?